utils/constants.js

```diff
@@ -0,0 +1,15 @@
+// You can use string format or `process.env` if you'd prefer
+const uuidv4 = require(`uuid/v4`);
+const pluginOptions = require(`../../../gatsby-config`);
+const ci = require('ci-info');
+const {plugins} = pluginOptions;
+let PLUGIN_OPTIONS = plugins.find(plugin => plugin.resolve === 'gatsby-plugin-newrelic-test').options;
+PLUGIN_OPTIONS.buildId = process.gatsbyTelemetrySessionId || uuidv4();
```

Comment:
As I mentioned before, I think it's unwise to extend the plugin options here. I think we should _either_ require these fields _or_ provide the default values at the time we're relying on these values. By defining them here, we now need to keep track of them in three places: the config file, the constants file, and the place where they're actually used.

Reply:
The reason I added the `buildId` here is because it needs to be a constant value throughout the plugin every time it's run, so I couldn't think of a better way to do that. If we generate this in the `gatsby-config.js` that will require asking the user to add this which i didn't want to do, but open to suggestions